api/orchestration.go

```diff
@@ -186,9 +186,15 @@ func (o *docDBOrchestrator) documentDBDetails(ctx context.Context, name string)
 		return nil, apierror.New(apierror.ErrNotFound, "cluster not found in our org", nil)
 	}
 
+	instances, err := o.docdbClient.GetDocDBInstances(ctx, name)
```

Comment:
I'm not sure why we didn't have this in the response originally. Did you need it for something specifically? Can you add the output of `GET /v1/docdb/{account}/{name}` (with the Instances) to the README, so I can see what it look like?